### ISSUE TITLE
XCFramework generation issue with Kotlin 1.6.20-RC

### ISSUE THREAD

**Issue by r4phab** [icerockdev/moko-resources#315]
I'm trying tu update my Kotlin Multiplatform library with Kotlin 1.6.20-RC but an error occurs when trying to build using the new build task : https://youtrack.jetbrains.com/issue/KT-50904

Here is the error happening when trying to build my XCFramework : 

```
Caused by: java.lang.ClassCastException: class org.jetbrains.kotlin.gradle.tasks.FrameworkDescriptor cannot be cast to class org.jetbrains.kotlin.gradle.plugin.mpp.Framework (org.jetbrains.kotlin.gradle.tasks.FrameworkDescriptor and org.jetbrains.kotlin.gradle.plugin.mpp.Framework are in unnamed module of loader org.gradle.internal.classloader.VisitableURLClassLoader @61a54c86)
        at dev.icerock.gradle.generator.apple.AppleMRGenerator$setupFatFrameworkTasks$fatAction$1.execute(AppleMRGenerator.kt:308)
        at dev.icerock.gradle.generator.apple.AppleMRGenerator$setupFatFrameworkTasks$fatAction$1.execute(AppleMRGenerator.kt:305)
```
        
Can you please provide a working version for Kotlin 1.6.20-RC ? 🙏

**Comment by Alex009** [comment]
will be available in 0.19.0

**Comment by r4phab** [comment]
Thank you for the fix, have a nice day ! 😬
